build.gradle

```diff
@@ -1,4 +1,4 @@
-plugins {
+;plugins {
```

Comment:
@raghav-codes / @404-bug-not-found did you notice this semi-colon? It clearly doesn't fail the build, but seems odd.